core/src/com/unciv/models/ruleset/unique/IHasUniques.kt

```diff
@@ -37,9 +43,89 @@ interface IHasUniques : INamed {
     fun getMatchingUniques(uniqueType: UniqueType, stateForConditionals: StateForConditionals? = null) =
         getMatchingUniques(uniqueType.placeholderText, stateForConditionals)
 
+    fun requiredTechs(): Sequence<String> {
+        val uniquesForWhenThisIsAvailable: Sequence<Unique> = getMatchingUniques(UniqueType.OnlyAvailableWhen, StateForConditionals.IgnoreConditionals)
+        if (uniquesForWhenThisIsAvailable.none() && uniques.any{ it.contains("Only available <after discovering [") })
```

Comment:
Wait... What? There's a simpler way to find conditionals. Uniques have a `conditionals` field so you can just do ``&& uniquesForAvailable.conditionals.any { it.isOfType(UniqueType.ConditionalTech }``